index.html

```diff
@@ -46,7 +46,182 @@ <h1 class="title playfair" >Pritikant Dhal</h1>
+</ul>
+<button class="project-btn flex">
+  <span>See this project</span>
+  <img
+     src="images/icons/ic_arrow_right.svg"
+     width="20px"
+     height="15px"
+     alt=""
+     srcset="" /> 
+</button>
+<div class="space">&nbsp;</div>
+</article>
+<article class="project flex">
+  <div>
+  <div class="bg3"></div>
+  </div> 
+<p>Project name goes here</p>
+<ul class="project-languages flex">
+  <li class="project-language">HTML/CSS</li>
+  <li class="project-language">Ruby on Rails</li>
+  <li class="project-language">JavaScript</li>
+</ul>
+<button class="project-btn flex">
+  <span>See this project</span>
+  <img
+     src="images/icons/ic_arrow_right.svg"
+     width="20px"
+     height="15px"
+     alt=""
+     srcset="" /> 
+</button>
+<div class="space">&nbsp;</div>
+</article>
+<article class="project flex">
+  <div>
+  <div class="bg4"></div>
+  </div> 
+<p>Project name goes here</p>
+<ul class="project-languages flex">
+  <li class="project-language">HTML/CSS</li>
+  <li class="project-language">Ruby on Rails</li>
+  <li class="project-language">JavaScript</li>
+</ul>
+<button class="project-btn flex">
+  <span>See this project</span>
+  <img
+     src="images/icons/ic_arrow_right.svg"
+     width="20px"
+     height="15px"
+     alt=""
+     srcset="" /> 
+</button>
+<div class="space">&nbsp;</div>
+</article>
+<article class="project flex">
+  <div>
+  <div class="bg5"></div>
+  </div> 
+<p>Project name goes here</p>
+<ul class="project-languages flex">
+  <li class="project-language">HTML/CSS</li>
+  <li class="project-language">Ruby on Rails</li>
+  <li class="project-language">JavaScript</li>
+</ul>
+<button class="project-btn flex">
+  <span>See this project</span>
+  <img
+     src="images/icons/ic_arrow_right.svg"
+     width="20px"
+     height="15px"
+     alt=""
+     srcset="" /> 
+</button>
+<div class="space">&nbsp;</div>
+</article>
+<article class="project flex">
+  <div>
+  <div class="bg6"></div>
+  </div> 
+<p>Project name goes here</p>
+<ul class="project-languages flex">
+  <li class="project-language">HTML/CSS</li>
+  <li class="project-language">Ruby on Rails</li>
+  <li class="project-language">JavaScript</li>
+</ul>
+<button class="project-btn flex">
+  <span>See this project</span>
+  <img
+     src="images/icons/ic_arrow_right.svg"
+     width="20px"
+     height="15px"
+     alt=""
+     srcset="" /> 
+</button>
+<div class="space">&nbsp;</div>
+</article>
+</section>
+
+<!--About Me Section-->
+<section class="about flex">
+  <h3 class="title popins">About</h3>
+  <p id="about-me" class="popins">
+    I can help you build a product , feature or website Look through some of 
+    my work and experience! If you like what you see and have a project you need coded,
+    don’t hesitate to contact me.
+  </p>
+  <button id="get-cv-btn" class="btn class popins">
+    <span>Get my CV</span>
+    <img src="images/icons/ic_download.svg" width="24" height="24" alt="" />
```

Comment:
- [ ] To adhere to [HTML and CSS best practices,](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) it is advisable to incorporate alt text for accessibility purposes and utilize an external CSS file.🥱🥱

- [ ] By including alt text for accessibility and utilizing an external CSS file, you adhere to best practices and improve the overall quality of your code.